### PR TITLE
Mark input purposes list as non-normative

### LINKED ISSUE #3777: Input Purposes for User Interface Components is an appendix, so should be marked as non-normative
Discussed in the backlog task force meeting on 5th April, [Input Purposes for User Interface Components](https://www.w3.org/TR/WCAG22/#input-purposes) is an appendix, so should be [marked as non-normative](https://www.w3.org/TR/WCAG22/#interpreting-normative-requirements).

The trick to this seems to be adding a `class="informative"` attribute to the containing `section` element, but building the site locally doesn't create the homepage, so can't check this.

### PR DESCRIPTION
**Pull request by fstrr**
Closes #3777

Building the site locally doesn't build a homepage, so I can't check to see if the newly-added class does what I think it's supposed to.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/pull/3778.html" title="Last updated on Apr 8, 2024, 5:15 PM UTC (65249fe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/3778/63d28b8...65249fe.html" title="Last updated on Apr 8, 2024, 5:15 PM UTC (65249fe)">Diff</a>

### CODE REVIEW

**Review by patrickhlauke — APPROVED** [review]

File: guidelines/input-purposes.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<section id="input-purposes">
+<section class="informative" id="input-purposes">
 	<h2>Input Purposes for User Interface Components</h2>
```
Comment (mbgower) [inline guidelines/input-purposes.html:2]:
```suggestion
	<h2>Appendix: Input Purposes for User Interface Components</h2>
```

**Review by scottaohara — APPROVED** [review]
approved on the assumption that Mike's suggestion will be accepted, or a separate "appendix" heading precedes this section (in the off chance there would need to be other items that should have been / could be added to the appendix)

**Comment by bruce-usab** [comment]
Discussed on TF call 4/12 and Ready for approval.  Might be impacted by resolution of #3539

**Comment by alastc** [comment]
Should we be using the [appendix class from respec](https://respec.org/docs/#appendix)? @iadawn presumably this would be an errata and need re-publishing to show?

**Comment by iadawn** [comment]
Yes, this would need to be republished to show.

There is something else to consider. Currently this is a section which is not marked as 'non-normative'. Based on [Interpreting Normative Requirements](https://www.w3.org/TR/WCAG/#interpreting-normative-requirements) this is normative. Changing it to an 'appendix' will make it non-normative.

**Comment by patrickhlauke** [comment]
@alastc @iadawn see https://github.com/w3c/wcag/pull/3778

**Comment by fstrr** [comment]
@iadawn see [this comment](https://github.com/w3c/wcag/pull/3539#issuecomment-2042989256)—that section is meant to be non-normative.

**Comment by alastc** [comment]
> ... Changing it to an 'appendix' will make it non-normative.

@iadawn it's something we lost between WCAG 2.0 and 2.1 (see [Mike's comment](https://github.com/w3c/wcag/issues/3777#issuecomment-2052610615)). So it means some sections were intended to be appendixes, but lost that marking.

**Comment by dbjorge** [comment]
Currently, 1.3.5 is normatively defined in terms of this seciton. I think that means we can't mark this section as informative unless we also rewrite the first bullet in SC 1.3.5.

**Comment by bruce-usab** [comment]
Discussed on TF call 9/13.  Closing as we need to track this one by the issue, not this PR.

**Pull request closed without merging**